### PR TITLE
Update settings.lua to the gamemode's config.lua

### PR DESCRIPTION
**Pull request by Down-s**
Just noticed a few things were different, I have copied it all over here.

### CODE REVIEW

File: lua/darkrp_config/settings.lua [hunk 1]
```diff
@@ -69,7 +69,7 @@ GM.Config.deadtalk                      = true
 GM.Config.deadvoice                     = true
 -- deathpov - Enable/disable whether people see their death in first person view.
 GM.Config.deathpov                      = false
--- decalcleaner - Enable/Disable clearing every player's decals.
+-- decalcleaner - Enable/Disable clearing ever players decals.
```
Comment (FPtje) [inline lua/darkrp_config/settings.lua:72]: This doesn't seem right. If this grammar error exists in DarkRP I think it's better to fix it there.

**Review by FPtje — CHANGES_REQUESTED** [review]
Thanks! Small comment

File: lua/darkrp_config/settings.lua (hunk 1 above)
Comment (FPtje) [reply lua/darkrp_config/settings.lua:72]: This is now fixed in DarkRP.

File: lua/darkrp_config/settings.lua [hunk 2]
```diff
@@ -69,7 +69,7 @@ GM.Config.deadtalk                      = true
 GM.Config.deadvoice                     = true
 -- deathpov - Enable/disable whether people see their death in first person view.
 GM.Config.deathpov                      = false
--- decalcleaner - Enable/Disable clearing every player's decals.
+-- decalcleaner - Enable/Disable clearing ever player's decals.
```
Comment (FPtje) [inline lua/darkrp_config/settings.lua:72]: Every* 🙈

**Review by FPtje — CHANGES_REQUESTED** [review]

File: lua/darkrp_config/settings.lua (hunk 2 above)
Comment (Down-s) [reply lua/darkrp_config/settings.lua]: Oh wow, don't know how I didn't see that. Thanks.

**Review by FPtje — APPROVED** [review]

**Comment by FPtje** [comment]
Thanks!

**Pull request merged**